### ISSUE TITLE
default clock module: cannot turn seconds off

### ISSUE THREAD

**Issue by mfread** [MichMich/MagicMirror#1106]
I cannot seem to turn off the seconds display in the default clock module when I add the config item displaySeconds: "false"

snapshot of a section from my ~/MagicMirror/config/config.js file
```
{
  module: "clock",
  position: "top_left",
  config: {
    timeFormat: "12",
    displaySeconds: "false",
    showPeriod: "true"
  }
},
etc
```

I am running Raspian Stretch on a Pi2B. I followed the installation instructions of MagicMirror from the main page of MichMich/MagicMirror on GitHub. MM v2.1.3

**Comment by yesyesuk** [comment]
Hi, 

one thing I noticed is that you have a single quote in the
following line when it should be a double quote: 

showPeriod: 'true"


It is the next parameter after the one you say doesn't work. Could be
that it fails parsing. 

Chris 

On 2017-12-28 10:02, mfread wrote: 

>
I cannot seem to turn off the seconds display in the default clock
module when I add the config item displaySeconds: "false" 
> 
> snapshot
of a section from my ~/MagicMirror/config/config.js file
> {
> module:
"clock",
> position: "top_left",
> config: {
> timeFormat: "12",
>
displaySeconds: "false",
> showPeriod: 'true"
> }
> },
> etc 
> 
> I am
running Raspian Stretch on a Pi2B. I followed the installation
instructions from the main page of MichMich/MagicMirror on GitHub. 
> 
>
--
> You are receiving this because you are subscribed to this thread.
>
Reply to this email directly, view it on GitHub [1], or mute the thread
[2].
 

Links:
------
[1]
https://github.com/MichMich/MagicMirror/issues/1106
[2]
